src/scala/scripts/TwitterScroogeGenerator.scala

```diff
@@ -53,12 +53,16 @@ class ScroogeGenerator extends Processor {
     // remote jars are jars that come from another repo, really no different from transitive
     val remoteSrcJars = getIdx(3)
 
+    // Remote jars that we should build scrooge classes for
```

Comment:
maybe lets expand this

Remote jars where we should export the generated class files built from that thrift. This is the remote jar version of `immediateThriftSrcJars`